sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -123,7 +124,11 @@ private void addBreadcrumb(final @NotNull Activity activity, final @NotNull Stri
       breadcrumb.setData("screen", getActivityName(activity));
       breadcrumb.setCategory("ui.lifecycle");
       breadcrumb.setLevel(SentryLevel.INFO);
-      hub.addBreadcrumb(breadcrumb);
+
+      final Map<String, Object> hintMap = new HashMap<>();
+      hintMap.put("Activity", activity);
```

Comment:
What about something less likely to be used by a user directly like:

```suggestion
      hintMap.put("Android:Activity", activity);
```

We could use such suffix in all our usages

Reply:
The idea is that users see it, and not the opposite, the SDK doesn't need it.
Similar to what's done on JS https://docs.sentry.io/platforms/javascript/configuration/filtering/#hints-for-breadcrumbs